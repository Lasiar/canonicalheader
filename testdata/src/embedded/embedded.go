package embedded

import "net/http"

type embeded struct {
	http.Header
}

func _() {
	embeded{}.Get("TT") // want `non-canonical header "TT", instead use: "Tt"`
}
