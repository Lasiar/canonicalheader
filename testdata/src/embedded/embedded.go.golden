package embedded

import "net/http"

type embeded struct {
	http.Header
}

func _() {
	embeded{}.Get("Tt") // want `non-canonical header "TT", instead use: "Tt"`
}
